### ISSUE TITLE
Create Subscription 400

### ISSUE THREAD

**Issue by JamesDullaghan** [peterkeen/payola#263]
I'm somewhat stuck... Have been for a few hours now and not making any progress, so maybe someone can help.

![screen shot 2016-10-04 at 6 33 06 pm](https://cloud.githubusercontent.com/assets/3254361/19097332/1df0f69a-8a61-11e6-9edc-df843b183d23.png)

Then a post to subscriptions happens, and responds with a "pending" status.

![screen shot 2016-10-04 at 6 35 54 pm](https://cloud.githubusercontent.com/assets/3254361/19097365/66c06540-8a61-11e6-84ee-3290ca0dd239.png)

After polling around 30 times, the same 400 response is received telling me my stripe token can't be used more than once?

![screen shot 2016-10-04 at 6 37 15 pm](https://cloud.githubusercontent.com/assets/3254361/19097387/994dfaea-8a61-11e6-875f-a56a039244ce.png)

Things I've tried:
`v1.3.0`
`v.1.4.0`
pointing to the github repo directly
pointed to local gem (logged js output for sanity)
`delayed_job` is definitely running and queuing the `Payola::Worker` job

The `plan_id` `stripeEmail` `plan_type` and `stripeToken` are being submitted to the `SubscriptionsController#create` action, not that it's relevant. Here's the implementation:

```ruby
  def create
    # set plans to fetched plan
    Rails.logger.info("Params : #{params}")
    Rails.logger.info("Fetched Plan : #{plan.inspect}")
    # => Fetched Plan : #<SubscriptionPlan id: 2, amount: 999, interval: "year", stripe_id: "yearly", name: "Yearly", interval_count: nil, trial_period_days: nil, created_at: "2016-10-04 18:01:43", updated_at: "2016-10-04 18:01:43">

    params[:plan] = plan
    new_subscription = Payola::CreateSubscription.call(params, current_user)
    render_payola_status(new_subscription)
  end
```

The `Payola::Subscription`'s are being created as well as the subscription, customer and card inside of stripe. Each of the `Payola::Subscription`'s have a unique `stripe_token`. This is kind of disheartening as the subscription in the app is being recorded as `errored` but in stripe it is `active`.

I've read through the source pretty thoroughly, as well as the provided wiki's and I'm banging my head here trying to figure out how the `stripe_token` has been used more than once. The overall implementation is pretty similar to the subscription wiki, but if you'd like to know anything specific, feel free to ask. If anyone can provide some insight, it would be greatly appreciated.

**Comment by JamesDullaghan** [comment]
Another odd thing I've noticed is the post request

**Comment by JamesDullaghan** [comment]
Make sure you check your delayed_job logs if you are logging delayed_job somewhere other than the development logs! After much headpounding, the issue was due to the fact I had moved the `payola.subscription.active` event to a service object, which was throwing an exception, but not being handled anywhere! The exception was being recorded in the delayed_job log, not the development log, causing much heartache!

The initial 402 error on the expiration date was caused by the load order of `jquery.payment` and `payola`.

6 hours later... We're in business!
